src/defargs.l

```diff
@@ -471,6 +471,7 @@ CPPC  "/\/"
                                            {
                                              yyextra->curTypeConstraint.resize(0);
                                              yyextra->lastExtendsContext=YY_START;
+                                             assert(yyextra->argSharpCount == 0);
```

Comment:
Not sure why an assert is necessary here? or does this have to do with my remark around the lines 503.? Wouldn't it be better to give a real doxygen warning?

Reply:
I believe ``argSharpCount != 0`` can never happen here. Just placed the ``assert` to see whether it would go through the ci. The line can be removed.